kafka-event-listener/src/main/java/org/hypertrace/core/kafka/event/listener/KafkaMockConsumerTestUtil.java

```diff
@@ -0,0 +1,57 @@
+package org.hypertrace.core.kafka.event.listener;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import lombok.Getter;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.MockConsumer;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import org.apache.kafka.common.Node;
+import org.apache.kafka.common.PartitionInfo;
+import org.apache.kafka.common.TopicPartition;
+
+public class KafkaMockConsumerTestUtil<K, V> {
```

Comment:
Test utils can go in testFixtures so that production classes do not depend on them. Please refer to the following for a sample test fixture and its usage. https://github.com/Traceableai/activity-event-service/tree/main/policy-store-service-impl/src/testFixtures

Reply:
Thanks, didn't know about them. Used them